docs/user_guide.md

```diff
@@ -1,4 +1,57 @@
-# CodeChecker
+Table of Contents
+=================
+* [CodeChecker](#codechecker)
+  * [Default configuration](#default-configuration)
+* [Easy analysis wrappers](#easy-analysis-wrappers)
+  * [`check`](#check)
+* [`PRODUCT_URL` format](#product-url-format)
+  * [Example](#product-url-format-example)
+* [Available CodeChecker commands](#available-commands)
+  * [`log`](#log)
+  * [`analyze`](#analyze)
+    * [_Skip_ file](#skip)
+      * [Example](#skip-example)
+    * [Analyzer configuration](#analyzer-configuration)
+      * [Compiler-specific include path and define detection (cross compilation)](#include-path)
+      * [Forwarding compiler options](#forwarding-compiler-options)
+        * [_Clang Static Analyzer_](#clang-static-analyzer)
+        * [_Clang-Tidy_](#clang-tidy)
+    * [Toggling checkers](#toggling-checkers)
+      * [Checker profiles](#checker-profiles)
+      * [`--enable-all`](#enable-all)
+    * [Cross Translation Unit (CTU) analysis mode](#ctu)
+  * [`parse`](#parse)
+    * [Suppression in the source code](#suppression-code)
+      * [Supported formats](#supported-formats)
+      * [Exporting source code suppression to suppress file](#suppress-file)
+  * [`store`](#store)
+    * [Using SQLite for database](#sqlite)
+  * [`checkers`](#checkers)
+  * [`analyzers`](#analyzers)
+  * [`server`](#server)
+    * [Creating a public server](#public-server)
+    * [Configuring database and server settings' location](#server-settings)
+    * [Master superuser and authentication forcing](#auth-force)
+    * [Enfore secure socket (SSL)](#ssl)
+    * [Managing running servers](#managing-running-servers)
+  * [cmd](#cmd)
+    * [List runs (`runs`)](#cmd-runs)
```

Comment:
I think we should change the order here for the subcommands.

`runs` (List of runs) 
`results` (Analysis results)
...

Reply:
The order is taken from how the help is generated by `argparse` which in turn, I think, takes the subcommands in the order they are defined in the source file.

Reply:
I meant not the order of the items (`runs`, `results`) but the order of the command name and the short description.

Reply:
Got it. That's a good idea. 😉